metricbeat/module/system/process/_meta/data.json

```diff
@@ -12,36 +12,209 @@
     "metricset": {
         "name": "process"
     },
+    "process": {
+        "args": [
+            "/usr/bin/dockerd",
+            "-H",
+            "unix://"
+        ],
+        "executable": "/usr/bin/dockerd-ce",
+        "name": "dockerd",
+        "pgid": 2080,
```

Comment:
And don't forget to leave group ID as text (wherever it ends up)

Reply:
By text I meant `keyword` datatype, here

Reply:
I agree that these ids could be `keyword`, but this is a value of the same kind as `pid` or `ppid` and they are defined as `long` in ECS :thinking:

Reply:
Gotcha, I was not familiar with `pgid`. If the semantics are like PID, this can be left as numeric, then.

So recap: I'm good with the field `process.pgid`, and I'm good with it remaining numeric 👍